tests_thien/utilities/constant.py

```diff
@@ -0,0 +1,22 @@
+# DA
+# http://192.168.188.99/TADashboard/login.jsp
+# http://192.168.189.60:54000/TADashboard/login.js
+DA_URL = "http://192.168.189.60:54000/TADashboard/login.jsp"
```

Comment:
Should have a way to update this variable in real-time. When I need to run all of the tests on other URL. What should I do?

Reply:
added environment module same with phi

Reply:
Sorry, added url in conftest